### PR TITLE
EPMRPP-64628: add frame for composite attribute

### PR DESCRIPTION
**Pull request by oranje322**

### CODE REVIEW

File: app/src/components/inputs/inputConditionalAttributes/inputConditionalAttributes.scss [hunk 1]
```diff
@@ -20,6 +20,16 @@
   min-height: 30px;
   box-sizing: border-box;
   background-color: $COLOR--white-two;
+  border: 1px solid $COLOR--gray-80;
+
+  &:hover {
+    border-color: $COLOR--gray-47;
+  }
+  &:focus-within {
```
Comment (chivekrodis) [inline app/src/components/inputs/inputConditionalAttributes/inputConditionalAttributes.scss:28]: `:focus-within` not supported by ie11

File: app/src/components/main/attributeList/editableAttribute/attributeEditor/attributeEditor.jsx [hunk 2]
```diff
@@ -173,6 +173,7 @@ export class AttributeEditor extends Component {
             />
           </FieldErrorHint>
         </div>
+        <span className={cx('separator')}>:</span>
```
Comment (chivekrodis) [inline app/src/components/main/attributeList/editableAttribute/attributeEditor/attributeEditor.jsx:176]: Now this separator applying for all places where we are using attribute editor. Does it expected behavior?

**Review by chivekrodis — CHANGES_REQUESTED** [review]

**Comment by chivekrodis** [comment]
Please recheck your implementation it is affected markup in all places where we are using attribute editor(border-bottom are missed, highlight on hover or focus state)
before 
image
after
image

File: app/src/components/inputs/inputConditionalAttributes/inputConditionalAttributes.scss [hunk 3]
```diff
@@ -17,9 +17,18 @@
 .input-conditional-attributes {
   position: relative;
   width: 100%;
-  min-height: 30px;
+  max-height: 30px;
```
Comment (AmsterGet) [inline app/src/components/inputs/inputConditionalAttributes/inputConditionalAttributes.scss:20]: May be just `height`?

File: app/src/components/main/attributeList/editableAttribute/attributeEditor/attributeEditor.jsx [hunk 4]
```diff
@@ -170,9 +180,11 @@ export class AttributeEditor extends Component {
               attributeKey={this.state.key}
               attributeValue={this.state.value}
               onInputChange={this.handleAttributeKeyInputChange}
+              borderView={borderView}
             />
           </FieldErrorHint>
         </div>
+        <span className={cx('separator')}>:</span>
```
Comment (AmsterGet) [inline app/src/components/main/attributeList/editableAttribute/attributeEditor/attributeEditor.jsx:187]: Can it be added via pseudoelement?

File: app/src/components/inputs/autocompletes/singleAutocomplete/singleAutocomplete.scss [hunk 5]
```diff
@@ -97,3 +97,7 @@
     }
   }
 }
+.border-view {
```
Comment (AmsterGet) [inline app/src/components/inputs/autocompletes/singleAutocomplete/singleAutocomplete.scss:100]: You are removing the `border` for border view? It looks contradictory.
May be better to call the property and the class f.e. `nakedView` or smth like that.

File: app/src/components/inputs/inputConditionalAttributes/inputConditionalAttributes.scss (hunk 1 above)
Comment (AmsterGet) [reply app/src/components/inputs/inputConditionalAttributes/inputConditionalAttributes.scss:28]: Decided to leave it as is (without IE support).
Discussed with @VolhaKarenko.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**